conda/psychopy-env.yml

```diff
@@ -4,9 +4,8 @@ channels:
 dependencies:
 - python=3.6
 - psychopy
-- pip
```

Comment:
Please leave this dependency in as new conda versions will emit a warning otherwise. (pip-installable deps listed but pip itself not included in the list of deps)

Reply:
Done. What about `pygame` - leave it in or remove?

Reply:
I would leave it in unless you know of any drawbacks this would cause?

Reply:
A potential drawback could be that at some point the package on PyPI fails to build on a specific platform. But right now it works, so this can be postponed until such a problem occurs.